<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationWidget</class>
 <widget class="QWidget" name="TransformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Transformation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Rotation:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxR</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>y:</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxSY</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" rowspan="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxSX">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x:</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxX</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxR">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Scale</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxSX</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Translation</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxX</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>y:</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxY</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxX">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxSY">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxY">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x:</string>
          </property>
          <property name="buddy">
           <cstring>doubleSpinBoxSX</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>55</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBoxX</tabstop>
  <tabstop>doubleSpinBoxY</tabstop>
  <tabstop>doubleSpinBoxSX</tabstop>
  <tabstop>doubleSpinBoxSY</tabstop>
  <tabstop>doubleSpinBoxR</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
